source/backend/plugin/CarlaPluginVST2.cpp

```diff
@@ -1301,7 +1302,28 @@ class CarlaPluginVST2 : public CarlaPlugin,
                     } // case kEngineControlEventTypeParameter
 
                     case kEngineControlEventTypeMidiBank:
-                        break;
+                        if ((pData->options & PLUGIN_OPTION_SEND_PROGRAM_CHANGES) != 0)
+                        {
+                            VstMidiEvent& vstMidiEvent_LSB(fMidiEvents[fMidiEventCount++]);
+                            carla_zeroStruct(vstMidiEvent_LSB);
+
+                            vstMidiEvent_LSB.type        = kVstMidiType;
+                            vstMidiEvent_LSB.byteSize    = kVstMidiEventSize;
+                            vstMidiEvent_LSB.deltaFrames = static_cast<int32_t>(isSampleAccurate ? startTime : event.time);
+                            vstMidiEvent_LSB.midiData[0] = char(MIDI_STATUS_CONTROL_CHANGE | (event.channel & MIDI_CHANNEL_BIT));
+                            vstMidiEvent_LSB.midiData[1] = MIDI_CONTROL_BANK_SELECT__LSB;
+                            vstMidiEvent_LSB.midiData[2] = char(ctrlEvent.value*127.0f);
```

Comment:
I followed the example on [line 1298](https://github.com/falkTX/Carla/pull/489/files/01957a23a1e61302764c0976bdfe3d415a530cac#diff-7f1b96f3c9ea7ee654b8b37a38764665R1298) there...

I think `ctrlEvent` is of type `EngineControlEvent` whose attribute "value" is "normalized between 0.0 and 1.0" according to CarlaEngine.hpp

So I think I've done the right thing?

Cheers